### PR TITLE
Various Bug Fixes

### LINKED ISSUE #10923: Morph airlock disguise has no sprite and therefore no way to remove disguise
## Round ID:
33842

## Reproduction:
I spawned in as a morph and changed into an airlock (named Port Quarter Solar Access). Upon doing so, my sprite totally disappeared, and I can no longer change back into my original form by any means, effectively ending my round.

### LINKED ISSUE #12029: when lazarus drakeling
## Test Merges: 
Not Supplied / None
## Reproduction:
When reviving a dead drakeling with lazarus their can_buckle does not get set to 1 preventing people from buckling
 Submitted by: Jamie (98856872550879232)

### LINKED ISSUE #12059: chaplain tentacle hand stuck to belt
## Test Merges: 
Not Supplied / None
## Reproduction:
The chaplains nullrod variant of the tentacle hand can be stored in the beltslot but doing this leaves it stuck to it
 Submitted by: cuackles (237761038500626442)

### LINKED ISSUE #12066: weird wacky gene description
## Test Merges: 
Not Supplied / None
## Reproduction:
.
 ## Supplied Image:

SuppliedImage
 Submitted by: cuackles (237761038500626442)

### PR DESCRIPTION
**Pull request by UselessTheremin**
# Github documenting your Pull Request

Fixes #12066 
Fixes #12059 
Fixes #10923
Ports https://github.com/BeeStation/BeeStation-Hornet/pull/3652

:cl:  
bugfix: you can no longer get the chaplain armblade stuck on your belt
bugfix: ash drake will now PROBABLY stay rideable once revived
bugfix: morphs can no longer turn into airlocks and become invisible godless homunculi  
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/drakeling.dm [hunk 1]
```diff
@@ -32,7 +32,7 @@
 	gold_core_spawnable = NO_SPAWN
 	butcher_results = list(/obj/item/stack/ore/diamond = 3) //you MONSTER
 	tame = TRUE
-	can_buckle = TRUE
+	can_buckle = 1
```
Comment (Bibby0110) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/drakeling.dm]: what, why, TRUE = 1

**Comment by SomeguyManperson** [comment]
problem with drakeling iirc is the riding datum gets deleted once it dies

**Comment by JamieD1** [comment]
Does not fix #12029

Removing from fix list until fixed.

**Review by adamsong — APPROVED** [review]
Fix works

**Pull request merged**